src/sentry/integrations/pipeline.py

```diff
@@ -40,6 +40,15 @@ def finish_pipeline(self):
         try:
             data = self.provider.build_integration(self.state.data)
         except IntegrationError as e:
+            self.get_logger().info(
+                'build-integration.failure',
+                extra={
+                    'error_message': e.message,
+                    'error_status': e.code,
+                    # TODO(lb): this looks like too much info... not sure how to limit it
+                    'data': self.state.data,
```

Comment:
This might also contain secrets like access tokens or private keys (jira-server). Might be best to omit this for now.